documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -91,11 +82,10 @@ Testing an RC:
     - [ ] Merge `release-vX.Y.Z` into the `releases` branch.
     - [ ] Tag this merge commit (on the `releases` branch) with `vX.Y.Z`
     - [ ] Cut the release [here](https://github.com/filecoin-project/lotus/releases/new?prerelease=true&target=releases).
-        - [ ]  Check `Create a discussion for this release` when tagging the release 
     - [ ] Final announcements
         - [ ] Update network.filecoin.io for mainnet, calib and nerpa.
-        - [ ] repost in #fil-lotus in filecoin slack
-        - [ ] Inform node provides (Protofire, Digital Ocean..)
+        - [ ] repost in #fil-lotus-announcement in filecoin slack
+        - [ ] Inform node providers (Protofire, Digital Ocean..)
```

Comment:
This isn't making things any "worse", but a couple of thoughts:
1. Put each item on its own line as I assume this is a different communication channel for each.
2. Ideally we validate when this gets done.  Per slack discussion, customers don't get benefit from a new Lotus release in Digital Ocean until Digital Ocean actually publishes the new image.